test/unit_tests/test_package_version.py

```diff
@@ -2,9 +2,10 @@
 import unittest
 import tempfile
 import shutil
+import mock
```

Comment:
Not in Python2 standard library, but this package already depends on `python-mock`, so 👍 .

Reply:
Yeah I checked for that and saw it in the package.xml so went with it.